penny_university/settings.py

```diff
@@ -24,7 +24,7 @@
 SECRET_KEY = '7e@1*$kub8yxjd&pkej#+0k+e9omlz!31$zuq(4$rglm4i(hp1'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get('PENNY_DEBUG', '').lower() == "true"
```

Comment:
🤕 oops!

Reply:
Good catch, but for this why can't you just do `os.eviron.get('PENNY_DEBUG', False)`?